### PR TITLE
add rudimentary sync; activate tests

### PR DESCRIPTION
**Pull request by JanPietrzyk**
@PascalThesing cant review here?!

### CODE REVIEW

File: src/Merchants/Content/Merchant/Services/CustomerSync.php [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                continue;
+            }
+
+            if(!$writeResult->getExistence()->exists()) {
+                $newCustomerId = Uuid::randomHex();
+
+                $customer[] = $saneDefaults;
+                $customer[] = [
+                    'id' => $newCustomerId,
+                    'customerNumber' => Uuid::randomHex(),
```
Comment (shyim) [inline src/Merchants/Content/Merchant/Services/CustomerSync.php]: Do we not need to save also the password to the customer. So we can login as him?

File: src/Merchants/Content/Merchant/Services/MerchantLogin.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    public function hijackLogin(CustomerLoginEvent $customerLoginEvent): void
+    {
+        $merchant = $this->fetchMerchant($customerLoginEvent);
+
+        if($merchant === null) {
+            return;
+        }
+
+        $this->
```
Comment (shyim) [inline src/Merchants/Content/Merchant/Services/MerchantLogin.php]: Syntax error :D

File: src/Merchants/Content/Merchant/Services/MerchantLogin.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+        dump($customerLoginEvent);
```
Comment (shyim) [inline src/Merchants/Content/Merchant/Services/MerchantLogin.php]: Dump?

Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/Services/MerchantLogin.php]: not active, in the next pr

File: src/Merchants/Content/Merchant/Services/MerchantLogin.php (hunk 2 above)
Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/Services/MerchantLogin.php]: not active, in the next pr

File: src/Merchants/Content/Merchant/Services/CustomerSync.php (hunk 1 above)
Comment (JanPietrzyk) [reply src/Merchants/Content/Merchant/Services/CustomerSync.php]: Maybe.... ;D Currently there is not test for that.

**Pull request merged**